pyproject.toml

```diff
@@ -31,7 +31,7 @@ build-backend = "setuptools.build_meta"
 
 [tool.black]
 line-length = 88
-target-version = ['py38', 'py39', 'py310']
+target-version = ['py39', 'py310']
```

Comment:
Not strictly related to this PR, but can we add `'py311'` to this list while we are here?

Reply:
you got it!